deploy/examples/cosi/bucketaccessclass.yaml

```
@@ -0,0 +1,9 @@
+kind: BucketAccessClass
+apiVersion: objectstorage.k8s.io/v1alpha1
+metadata:
+  name: sample-bac
+driverName: ceph.objectstorage.k8s.io
+authenticationType: KEY
+parameters:
+  objectStoreUserSecretName: rook-ceph-object-user-my-store-my-user
```

Doesn't this also need to be the cosi user?
```suggestion
  objectStoreUserSecretName: rook-ceph-object-user-my-store-cosi
```